test/pyth-tests/pyth.test.ts

```diff
@@ -1,59 +0,0 @@
-import 'dotenv/config';
```

Comment:
Pyth is still relevant with the new integration. Please uncomment and test everything related to Pyth